### PR TITLE
[MNG-7371] Parent POM for dependency packaging is not pom

### PR DESCRIPTION
**Pull request by cstamas**
The IT is related to parents POMs of ANYTHING (not current
project) that must have packaging POM.

https://issues.apache.org/jira/browse/MNG-7371

### CODE REVIEW

File: core-it-suite/src/test/resources/mng-7371-parent-pom-packaging-is-pom/pom-template.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    <packaging>jar</packaging>
+
+    <name>Maven Integration Test :: MNG-7371</name>
+    <description>
+        Test that every parent POM used everywhere has packaging POM
+    </description>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>1.8</maven.compiler.source>
```
Comment (michael-o) [inline core-it-suite/src/test/resources/mng-7371-parent-pom-packaging-is-pom/pom-template.xml:39]: This contradicts with Maven 3.6.1

File: core-it-suite/src/test/resources/mng-7371-parent-pom-packaging-is-pom/repo/org/apache/maven/its/mng7371/a/1/a-1.pom.sha1 [hunk 2]
```diff
@@ -1 +1 @@
-e4661f80f5265a892f65aad88613b39ac25b3265
\ No newline at end of file
+b20adf1573f3c241b1d5fec2fe2cd2ab6e305906  a-1.pom
```
Comment (michael-o) [inline core-it-suite/src/test/resources/mng-7371-parent-pom-packaging-is-pom/repo/org/apache/maven/its/mng7371/a/1/a-1.pom.sha1:1]: No, our files contain the hash only.

File: core-it-suite/src/test/resources/mng-7371-parent-pom-packaging-is-pom/repo/org/apache/maven/its/mng7371/b/1/b-1.pom.sha1 [hunk 3]
```diff
@@ -1 +1 @@
-d2c3318bde4e19f9b4d8aedd01bf2edf061f73b9
\ No newline at end of file
+3ea4ddbcc1801191fed2417187bd43cd2dde7417  b-1.pom
```
Comment (michael-o) [inline core-it-suite/src/test/resources/mng-7371-parent-pom-packaging-is-pom/repo/org/apache/maven/its/mng7371/b/1/b-1.pom.sha1:1]: Same here

File: core-it-suite/src/test/resources/mng-7371-parent-pom-packaging-is-pom/repo/org/apache/maven/its/mng7371/parent-b/1/parent-b-1.pom.sha1 [hunk 4]
```diff
@@ -0,0 +1 @@
+736997894840b66a9c4d93a7f2661b387b1590d1  parent-b-1.pom
```
Comment (michael-o) [inline core-it-suite/src/test/resources/mng-7371-parent-pom-packaging-is-pom/repo/org/apache/maven/its/mng7371/parent-b/1/parent-b-1.pom.sha1:1]: Same here

**Review by michael-o — CHANGES_REQUESTED** [review]

**Comment by cstamas** [comment]
This was a fluke, closing, dropping this

**Pull request closed without merging**